x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/apm/dependencies/generate_many_dependencies.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ApmSynthtraceEsClient } from '@kbn/synthtrace';
+import type { Instance } from '@kbn/synthtrace-client';
+import { apm, timerange } from '@kbn/synthtrace-client';
+
+const MAX_DEPENDENCIES = 10000;
+const MAX_DEPENDENCIES_PER_SERVICE = 500;
+const MAX_SERVICES = 20;
+
+export async function generateManyDependencies({
```

Comment:
~~Scout already exposes `synthtrace` fixture, won't it work in this case?~~

I just realised it is FTR API test, all good